contents/docs/libraries/ios/index.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: iOS
+title: iOS SDK installation
```

Comment:
Naming it this impact this page: https://posthog-git-split-ios-sdk-post-hog.vercel.app/docs/getting-started/install?tab=sdks